stubs/Authlib/METADATA.toml

```
@@ -1,4 +1,4 @@
-version = "~= 1.6.4"
+version = "~= 1.6.5"
```

Let's also use a fixed version to minimize the number of daily test failures.


```suggestion
version = "1.6.5"
```